package/com.unity.formats.usd/Runtime/Scripts/IO/Scene/SceneImporter.cs

```diff
@@ -252,6 +252,20 @@ public ImportException(string message, System.Exception innerException)
         throw new ImportException("Null USD Scene");
       }
 
+      // The matrix to change the handedness (the basis) is different for FBX and doesn't
+      // follow convention. This bit change this basisChange matrix to match the user options.
+      // [DEFAULT] SlowAndSafe is the standard and transform (X,Y,Z) basis to (X,Y,-Z)
+      // SlowAndSafe_AsFBX is consistent with FBX/Alembic/Obj ans transform (X,Y,Z) basis to (-X,Y,Z)
```

Comment:
Maybe:
```
The matrix to convert USD (right-handed) to Unity (left-handed) is different for the legacy FBX importer
and incorrectly swaps the X-axis rather than the Z-axis. This changes the basisChange matrix to match the
user options. See SceneImportOptions.BasisTransformation for additional details.
```

I think there is some way to literally reference that class (BasisTransformation) from the doc string above, but I forget how. It's like \<ref\> or something.